ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -232,8 +232,8 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity
-- as: azure-ipi-ovn-ipsec-arm-mixarch-f28
-  cron: 13 12 14 * *
+- as: azure-ipi-ovn-ipsec-arm-mixarch-day2-f28
```

Comment:
I don't see the job in `openshift-openshift-tests-private-release-4.14__multi-nightly.yaml`, could you please double check?

Reply:
https://github.com/openshift/release/pull/55329/files#r1708727047